### DISCUSSION TITLE
Change /meetings/ URL

### DISCUSSION THREAD

**Discussion by trodat216** [code4recovery/12-step-meeting-list#1205] [Q&A]
We've had the same URL to our meetings search page for about 15 years and I'm hoping we don't have to change it:

CURRENTLY: site.com/find-a-meeting/
to this
NEW URL: site.com/meetings/

It's the most popular page on our website. Is there a way to change the /meetings/ URL of the main page via a Permalink or .htaccess rewrite?

**Comment by anchovie91471** [comment]
By adding this to your theme's function.php file, it will change your /meetings/ page url:

$tsml_slug = 'find-a-meeting';

**Comment by trodat216** [comment]
I put the suggested code in my child theme functions.php file. But, when I go to the URL, it gives a 404.

**Reply by SIA-WEBSUP** [reply]
did you re-saving the permalinks as described here:
    - https://wordpress.org/plugins/12-step-meeting-list/#can%20i%20change%20the%20url%20of%20the%20meetings%20list%3F

**Comment by trodat216** [comment]
That seems to work, thank you! The only other issue I see with this transition, is when I click on "2 other meetings at this location" on the meeting detail page, it redirects me back to our homepage:

Link:
https://www.site.com/locations/mary-queen-of-peace/?tsml-day=3&tsml-region=cleveland&tsml-type=M

Redirect
https://www.site.com/?tsml-day=3&tsml-region=cleveland&tsml-type=M

Shouldn't this bring you to a list of other meetings?

**Reply by tim-rohrer** [reply]
Hi @trodat216,

TSML will bring up a list of meetings at the location. Something is causing the redirect. If you post the actual site address, we can closer. Otherwise, try looking in the debug console for the browser to see if any hints are provided.

Tim

**Reply by trodat216** [reply]
Hi Tim, it seems like the redirect is happening in at couple places. Thanks. The URL is: https://www.aacle.org

**Reply by SIA-WEBSUP** [reply]
On most browsers, if you hover over a hyperlink, you can see the URL it will send you to on the lower left hand corner of the browser window.

On your website, if you go to Lakeside Dawn ( https://www.aacle.org/find-a-meeting/lakeside-dawn/ )and hover over the panel with the location, you'll note that it is linked correctly to ( https://www.aacle.org/locations/bay-presbyterian-church/ ).

Now we just need to figure out why this is being redirected to your homepage. My guess is that all 404s ( page not found ) are being redirected to your homepage.

For example, something like https://www.aacle.org/boo will be redirected to your homepage.

So the next question is why ( https://www.aacle.org/locations/bay-presbyterian-church/ ) is not being found...

I'm a little new to responding to these questions... but I was wondering if folks are in the habit of allowing others ( like me ) access to your wordpress backend to take a look at what might be going on...

**Reply by tim-rohrer** [reply]
@trodat216, 

It does seem something is conflicting on your site with the the page that displays location information. I've not seen this occur before specific to this page. 

In the console for your site, I see an error about `stats.wp.com`. I don't know where that is coming from (someone else here might be more familiar with it) but perhaps do what is needed to clear that error and see if the situation resolves. One trick often followed is to disable all your plugins except TSML and see if the misbehavior goes away. If it does, then start adding the plugins back in one at a time until the error returns.

**Reply by SIA-WEBSUP** [reply]
Two things you might try..

1. go back and roll back the change you made to $tsml_slug ... I don't suppose that is the issue, but it would be good to rule it out.
2. try switching to the new TSML UI interface. please report back if this 'solves' the problem. ( i say 'solve' because it won't be solving anything, just avoiding the problem )

**Reply by trodat216** [reply]
Thanks for your feedback. Sounds like we'll have to troubleshoot. Unfortunately, I can't just turn off plugins to test for conflicts as we have a lot of customizations done on the backend. I did switch to the TSML UI layout and it has a different way of listing all the meetings at the same location which works. But, the style will need some work to properly integrate into our layout. I did notice that if I go to: https://www.aacle.org/locations/ it does NOT redirect and I can scroll through all locations. The redirect only happens when I go to a specific location.
